### ISSUE TITLE
JSON error if multiple local addresses

### ISSUE THREAD

**Issue by arthurio** [alexchantastic/alfred-ip-address-workflow#1]
I have multiple local addresses, the debug console gives me this:

```
[2018-08-20 13:20:24][ERROR: input.scriptfilter] JSON error: Unescaped control character around character 54. in JSON:
{"items": [

	{
		"title": "Local IPv4: 192.168.141.91
192.168.143.155
169.254.241.90",
		"subtitle": "Press enter to paste or ⌘C to copy",
		"arg": "192.168.141.91
192.168.143.155
169.254.241.90"
	},

	{
		"title": "External IPv4: xxx.xxx.xxx.xxx",
		"subtitle": "Press enter to paste or ⌘C to copy",
		"arg": "xxx.xxx.xxx.xxx"
	},

	{
		"title": "Local IPv6: fe80::c6f:4428:7e31:5e23",
		"subtitle": "Press enter to paste or ⌘C to copy",
		"arg": "fe80::c6f:4428:7e31:5e23"
	},

	{
		"title": "External IPv6: xxx.xxx.xxx.xxx",
		"subtitle": "Press enter to paste or ⌘C to copy",
		"arg": "xxx.xxx.xxx.xxx"
	}

]}
```
Probably need to replace `\r\n` with `, `?

**Comment by arthurio** [comment]
Maybe append `| tr -s '\r\n' ',' | sed -e 's/,$//'` to the end of the grep for local ipv4 and ipv6?

**Comment by ITRecords38** [comment]
Hi,

Same problem also.

Starting debug for 'IP Address'

[2018-10-10 16:17:32][ERROR: input.scriptfilter] JSON error: Unescaped control character around character 48. in JSON:
{"items": [

	{
		"title": "Local IPv4: 0.0.0.15
192.168.1.13",
		"subtitle": "Press enter to paste or ⌘C to copy",
		"arg": "0.0.0.15
192.168.1.13"
	},

	{
		"title": "External IPv4: ",
		"subtitle": "Press enter to paste or ⌘C to copy",
		"arg": ""
	},

	{
		"title": "Local IPv6: ",
		"subtitle": "Press enter to paste or ⌘C to copy",
		"arg": ""
	},

	{
		"title": "External IPv6: ",
		"subtitle": "Press enter to paste or ⌘C to copy",
		"arg": ""
	}

]}

**Comment by alexchantastic** [comment]
Apologies on the slowness on this! I had GitHub notifications disabled for some reason on this repo 🤦‍♂️ .

@roshanca's fix will be released as part of the v1.0.1 later today.

**Comment by alexchantastic** [comment]
I just released [v1.0.1](https://github.com/alexchantastic/alfred-ip-address-workflow/releases/latest). Let me know if the problem is fixed.
